### PR TITLE
Bump style guide to 160.0.0 and add mobile icons

### PR DESCRIPTION
**Pull request by matzimowski**

### CODE REVIEW

File: style-guide/index.jsx [hunk 1]
```diff
@@ -31,14 +32,15 @@ const IndexPage = () =>
...
       <LogoPage />
       <AvatarsPage />
       <BoxesPage />
       <BubblesPage />
       <ButtonsPage />
       <CheckboxesPage />
-      <CountersPage />
-      <DropdownsPage />
+      <CountersPage /> */}
+      {/* <DropdownsPage /> */}
```
Comment (bartoszlorek) [inline style-guide/index.jsx:43]: what the dropdowns!?

Comment (matzimowski) [reply style-guide/index.jsx:43]: hah, yea, we need to take care of them separately, that are broken now here after last updates in sg

Comment (bartoszlorek) [reply style-guide/index.jsx:43]: what about `*/}` after CountersPage?

Comment (matzimowski) [reply style-guide/index.jsx:43]: good catch!

Comment (matzimowski) [reply style-guide/index.jsx]: done

**Pull request merged**